src/Werte/Dokumente/Dokumenttypen/Zertifikat.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Demv\Werte\Dokumente\Dokumenttypen;
+
+use Demv\Werte\Dokumente\DokumenttypInterface;
+use Demv\Werte\Value;
+
+/**
+ * Class Erstantrag
+ * @package Demv\Werte\Dokumente\DokumentTypen
+ */
+final class Zertifikat extends Value implements DokumenttypInterface
+{
+    public const ID = 598;
```

Comment:
Hast du das Zertifikat in der master Datenbank händisch angelegt und hier einfach die Id eingetragen?